rebasehelper/checkers/csmock_tool.py

```diff
@@ -22,7 +22,7 @@
 
 from __future__ import print_function
 import os
-from six import StringIO
+import six
```

Comment:
You should also add a blank line before the import (`six` is third-party library).